CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * [#7882](https://github.com/rubocop-hq/rubocop/pull/7882): Fix `Style/CaseEquality` when `AllowOnConstant` is `true` and the method receiver is implicit. ([@rafaelfranca][])
 * [#7790](https://github.com/rubocop-hq/rubocop/issues/7790): Fix `--parallel` and `--ignore-parent-exclusion` combination. ([@jonas054][])
 * [#7881](https://github.com/rubocop-hq/rubocop/issues/7881): Fix `--parallel` and `--force-default-config` combination. ([@jonas054][])
+* [#7635](https://github.com/rubocop-hq/rubocop/issues/7635): Fix a false positive for `Style/MultilineWhenThen` when a `when` body has multiple lines with breakline. ([@koic][])
```

Comment:
I don't quite get "multiple lines with breakline". Don't you always have line breaks in something that's multiline?

Reply:
Ah... this is a strange sentence 💦 I updated the sentence to:
"when `then` required for a body of `when` is used."